source/sass_api_reference/index.html.md.erb

```diff
@@ -0,0 +1,73 @@
+---
+title: Sass API Reference
+weight: 65
+---
+
+# Sass API Reference
+
+<% format_sassdoc_data(data).each do |heading, group| %>
+
+## <%= group_heading(heading) %>
+
+<% group.each do |doc| %>
+
+### <%= doc_heading(doc) %>
+
+<%= doc.description %>
+
+<% if doc.parameter %>
+| Name | Description | Type | Default value |
+| ---- | ----------- | ---- | ------------- |
+<% parameters_table(doc.parameter).each do |param| %>
+|  <%= param.name %> | <%= param.description %> | <%= param.type %> | <%= param.default_value %> |
+<% end %>
+<% end %>
+
+#### Usage
+
+<% if doc.deprecated %>
+    <% partial(:warning_text) do %>
+        <%= markdown(doc.deprecated) %>
+    <% end %>
+<% end %>
+
+<% if doc.parameter %>
+    <% inline_parameters = "(#{inline_parameters(doc.parameter)})" %>
+<% end %>
+
+<% if doc.context.type == 'mixin' %>
+```scss
+@include <%= doc.context.name %><%= inline_parameters %><%= mixin_trailing_code(doc.context.code) %>
```

Comment:
I found this a bit confusing at first because you've used the same name for the variable as you have for the helper, so at first glance I thought this was calling the helper without any parameters.

I _think_ it might also cause problems if we wanted to use the inline_parameters again in the code, as I assume the variable's existence means there's no longer a way to call it?